alf/experience_replayers/replay_buffer.py

```diff
@@ -535,7 +549,15 @@ def _compute_store_episodic_return(self, env_ids):
         # TODO: bootstrapping the the episodic return with the default_return
         # if game ends with timelimit. For example, the episode length of
         # Atari games is limited to 4500, which can be reached quite often.
-        first_step_pos = self.get_episode_begin_position(current_pos, env_ids)
+
+        # This is for computing from episode start, which shouldn't be used
+        # when computing episodic return every time discount = 0 step is added:
+        # first_step_pos = self.get_episode_begin_position(current_pos, env_ids)
```

Comment:
Is this comment still valid?

Reply:
It's valid.  Let me remove this since it's no longer used.